File: C programming/Competitive/Assignments/Assignment_11/Que_3.c
//question 3: Write a program which accepts starting and ending point from user and return addition of all numbers in between that range.

#include<stdio.h>

int RangeSum(int iStart , int iEnd)
{
    int iSum = 0;

    if(iStart > iEnd || iStart < 0 || iEnd < 0)
    {
        return -1;   // indicates invalid range
    }

    for(int i = iStart; i <= iEnd; i++)
    {
        iSum += i;
    }
    return iSum;
}

int main()
{
    int iValue1 = 0, iValue2 = 0, iRet = 0;

    printf("Enter starting point: ");
    scanf("%d",&iValue1);

    printf("Enter ending point: ");
    scanf("%d",&iValue2);

    iRet = RangeSum(iValue1, iValue2);

    if(iRet == -1)
    {
        printf("Invalid range\n");
    }
    else
    {
        printf("Addition is: %d\n", iRet);
    }

    return 0;
}

File: Java Programming/Competitve/Assignments/Asssignment_18/Que_3.java
//Write a Program to print all odd numbers up to N

class Logic
 {
    void printOddNumbers(int n) 
    {
        for (int i = 1; i <= n; i++) 
        {
            if (i % 2 != 0) 
            {
                System.out.print(i + " ");
            }
        }
        System.out.println();
    }
}


class Que_3
{
    public static void main(String A[])
    {
        Logic obj = new Logic();
        obj.printOddNumbers(20);
    } 
}